integration_tests/snapshot-serializers/transformer.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+module.exports = {
+  process(src, filename, config, options) {
+    if (/bar.js$/.test(filename)) {
+      return `${src};\module.exports = createPlugin('bar');`;
```

Comment:
I think the backslash (`\`) is a little confusing here at the first look. Can we remove it?